packages/launcher/src/renderer/launcher/OnboardIdentityView.js

```diff
@@ -1,13 +1,15 @@
 //@flow
 
 import React, { Component } from 'react'
-import { View, StyleSheet, ActivityIndicator } from 'react-native-web'
+import { ActivityIndicator } from 'react-native'
+
+import { Button, TextField, Row, Column, Text } from '@morpheus-ui/core'
+import { CircleArrowRight } from '@morpheus-ui/icons'
```

Comment:
It's not an issue for the launcher as the JS bundle size doesn't matter much, but for apps we'll have to be careful about only importing the code that is needed.
Material UI makes icon imports explicit rather than providing them all at once and I think this is a good approach to prevent this kind of issue, any way we could expose icons and import them as `import CircleArrowRight from '@morpheus-ui/icons/CircleArrowRight'` for example?

Reply:
done.